serving/samples/helloworld-dart/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM google/dart-runtime
+
+# google/dart-runtime expects us to listen on port 8080
+ENV PORT 8080
```

Comment:
This is not needed, Knative injects a PORT env var with the appropriate value.

Reply:
Sweet :)